Resources/Private/Language/locallang_csh_media.xlf

```diff
@@ -6,8 +6,14 @@
 			<trans-unit id="type.description" xml:space="preserve">
 				<source>The type of the media element, can be video or audio.</source>
 			</trans-unit>
+			<trans-unit id="subtitles.description" xml:space="preserve">
+				<source>An optional subtitle file(s) for the video.</source>
+			</trans-unit>
+			<trans-unit id="language.description" xml:space="preserve">
+				<source>Please enter the subtitle language in ISO 639-1 format.</source>
```

Comment:
```suggestion
				<source>Please enter the subtitle language in ISO 639-1 format (e. g. „de“, „en“, „fr“).</source>
```

Reply:
I thought about it, but was not so sure! Pointing out possibilities is always a good idea, thanks!